src/main/java/catan/domain/model/game/GameUserBean.java

```diff
@@ -43,6 +42,9 @@
     @Column(name = "MOVE_ORDER", unique = false, nullable = false)
     private int moveOrder;
 
+    @Embedded
+    private ResourcesBean resources = new ResourcesBean();
```

Comment:
we don't need to create new instance of ResourcesBean here since we create it in constructor